numpy/_core/defchararray.py

```diff
@@ -969,6 +1026,16 @@ def isspace(a):
     See Also
     --------
     str.isspace
+
+    Examples
+    --------
+    >>> str = "GHC"
```

Comment:
I did not check other existing docstrings, but do we need to create the intermediate variable 'str' here? Also `str` is a python builtin

Reply:
You are right. I forgot and didn't get a warning because I was editing the docstrings. I will remove these instances and make everything one liners.